.circleci/config.yml

```diff
@@ -584,6 +584,20 @@ jobs:
           environment:
             NODE_ENV: test
 
+  integration-test-libs:
+    executor: fullstack-executor
```

Comment:
Herein lies the difference for e.g. DB and other infra dependencies for these integration tests (that and `wait-for-infrastructure` further down).